conda_build_all/builder.py

```diff
@@ -34,8 +36,7 @@
 
 def package_built_name(package, root_dir):
     package_dir = os.path.join(root_dir, package)
-    meta = MetaData(package_dir)
-    return bldpkg_path(meta)
+    return conda_build.api.get_output_file_path(package_dir)
```

Comment:
Incidentally, if that is true, it also suggests that this function hasn't previously been tested 😱